src/handlers/lib/utility.js

```diff
@@ -513,6 +513,7 @@ const proceed = async (params, opts) => {
   if (fromSwitch) {
     message.value.to = message.value.from
     message.value.from = Enum.headers.FSPIOP.SWITCH
+    delete message.value.content.headers.accept
```

Comment:
Hi. This is handled l by the transformer in ml-api. Please remove it from here.

Reply:
fixed